timeless/items/models.py

```diff
@@ -21,7 +18,36 @@ class Item(DB.Model):
     created_on = DB.Column(DB.DateTime, default=datetime.utcnow, nullable=False)
     updated_on = DB.Column(DB.DateTime, onupdate=datetime.utcnow)
     company = DB.relationship("Company", back_populates="items")
+    employee_id = DB.Column(DB.Integer, DB.ForeignKey("employees.id"))
+    empolyee = DB.relationship("Employee")
+
+    def assignTo(self, employee):
```

Comment:
@ammaratef45 in python usually use snake case `def assign_to` or just `def assign` - good to me too